sr_hand/launch/gazebo/loaders/hand_controllers.launch

```diff
@@ -1,9 +1,12 @@
 <launch>
   <arg name="hand_id" default="rh" />
+  <arg name="hand_serial" default="1234"/>
   <arg name="hand_controller_gazebo"
   default="$(find sr_description)/hand/config/$(arg hand_id)_controller_gazebo.yaml" />
   <arg name="hand_effort_controller_gazebo"
   default="$(find sr_description)/hand/config/$(arg hand_id)_effort_controller_gazebo.yaml" />
+  <param name="/hand/mapping/$(arg hand_serial)" value="$(arg hand_id)"/>
+  <param name="/hand/joint_prefix/$(arg hand_serial)" value="$(arg hand_id)_"/>
```

Comment:
This parameter is loaded here and also in sr_hand/launch/gazebo/gazebo_hand.launch. Shouldn't it be only in one place?